redis/simpleresource.go

```diff
@@ -0,0 +1,86 @@
+package redis
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/datatrails/go-datatrails-common/logger"
+	otrace "github.com/opentracing/opentracing-go"
+)
```

Comment:
I wouldn't want to hold up the PR over this. But calling things 'Simple' anything has always grated with me. They are simple relative to what ? And now I have lost a noun that could actually have been descriptive :-)

Reply:
I think we loosely planned to call this something like jsonResource, as we are storing json strings?

Reply:
To be fair, this probably needs to be JsonResource or similar anyway - good spot